BankSDK/GiniBankSDKPinning/README.md

```diff
@@ -23,17 +23,21 @@ Further documentation with installation, integration or customization guides can
 
 ## Example
 
-In order to run [the example app](https://github.com/gini/gini-mobile-ios/tree/main/BankSDK/GiniBankSDKExample/GiniBankSDKExample), clone the repo ,open the project file and Resolve Package Versions in Xcode `File->Packages->Resolve Package Versions`.
+In order to run [the example app](https://github.com/gini/gini-mobile-ios/tree/main/BankSDK/GiniBankSDKExample/GiniBankSDKExample), clone the repo ,open the project file and Resolve Package Versions in Xcode `File->Packages->Resolve Package Versions` and inject your API credentials in 
```

Comment:
You missed this one: https://github.com/gini/gini-mobile-ios/pull/15#discussion_r749548064 👀

Reply:
🙈 Thank you.